Documentation/BasicUsage.md

```diff
@@ -118,7 +118,7 @@ Cycling all of the tracks and stopping them is more than enough to clean up afte
 You won't usually need to do this for remote tracks, only local.  
 
 ```javascript
-localMediaStream.getTracks().map(
+localMediaStream.getTracks().forEach(
```

Comment:
This works the same, slight speed difference?

Reply:
Semantics mainly. I would expect some return from `map` method. But I agree that it's not that crucial :)